components/gripper.py

```diff
@@ -23,7 +25,9 @@ def __init__(self) -> None:
         )
 
     def open(self) -> None:
+
         self.opened = True
+        self.open_time = time.monotonic()
```

Comment:
I think you will need to check if the claw is already open like what's done in close()